CMakeLists.txt

```diff
@@ -3470,6 +3444,54 @@ if(FFMPEG)
   target_include_directories(mixxx-lib PUBLIC "${FFMPEG_INCLUDE_DIRS}")
 endif()
 
+# STEM file support
+default_option(STEM "STEM file support" "FFMPEG_FOUND;FFMPEG")
+if (STEM)
+  if(NOT FFMPEG)
+    message(FATAL_ERROR "STEM requires that also FFMPEG is enabled")
+  endif()
+  target_compile_definitions(mixxx-lib PUBLIC __STEM__)
+  target_compile_definitions(mixxx-test PUBLIC __STEM__)
+  target_sources(mixxx-test PUBLIC
+    src/test/stemtest.cpp
+    src/test/steminfotest.cpp
+  )
+  target_precompile_headers(mixxx-lib PUBLIC
+    src/track/steminfo.h
+  )
```

Comment:
I guess only one `target_precompile_headers` statement per target is possible. Try to add the file as entry after line 1334 encapsulated in `if (STEM)` `endif()`

Reply:
Looks like Windows is still not happy. Not sure why only `soundsource.cpp` has unresolved symbol. Or does MSVC compiler only display the first object missing symbols?